### ISSUE TITLE
gnr_resolve bug: some names return just genus name from API

### ISSUE THREAD

**Issue by sckott** [ropensci/taxize#569]
via @noamross

egs of this problem with IUCN and ITIS datasources

an ITIS example:
- for some names, e.g. _Alouatta guariba_ the `name_string` and `canonical_form` fields have only the genus name without the epithet  http://resolver.globalnames.org/name_resolvers.json?names=Alouatta%20guariba&data_source_ids=3
- while for other names, e.g., _Helianthus annuus_ the `name_string` and `canonical_form` fields have the full genus + epithet names   http://resolver.globalnames.org/name_resolvers.json?names=Helianthus%20annuus&data_source_ids=3

noam said that he's seen this happen with at least IUCN as well

**Comment by sckott** [comment]
@dimus any thoughts on this? perhaps the epithet in some cases is being lost on some regex operation?

**Comment by dimus** [comment]
@sckott it means no dataset knows about this species, so it returns genus only. It is indicated in a admittedly cryptic manner by "match_type 6" . Match types are described here -- http://resolver.globalnames.org/about

In new resolver which is going to be released in December we will have match types in plain English

**Comment by sckott** [comment]
thanks for the quick response @dimus 

however, i'm not sure i understand. _Alouatta guariba_ is a valid name in ITIS, see https://www.itis.gov/servlet/SingleRpt/SingleRpt?search_topic=TSN&search_value=944185#null

so why would we only get back the genus?

**Comment by dimus** [comment]
Yes, you are right @sckott, I made a fresh import of ITIS, now http://resolver.globalnames.org/name_resolvers.json?names=Alouatta%20guariba&data_source_ids=3 is found

**Comment by sckott** [comment]
@dimus thanks much!  

are the `updated_at` dates here http://resolver.globalnames.org/data_sources.json updated and correct?  that is, could we use those to determine if a data source may be somewhat out of date?

**Comment by dimus** [comment]
@sckott @noamross I made an update to json output --- now there is `imported_at` field which shows when returned record was inserted into GN database, and there is `match_value` which explains a match type in English

**Comment by sckott** [comment]
thanks @dimus , looks good
